### PR TITLE
[General] Just replaced the code for deleting an application with an …

### PR DESCRIPTION
**Pull request by partik03**
…already existing function

Signed-off-by: Partik SIngh <partik.sbumrah.cse21@itbhu.ac.in>

**Notes for Reviewers**

This PR fixes N.A

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by partik03** [comment]
@leecalcote @Chadha93 Please review this pr

**Review by Abhishek-kumar09 — DISMISSED** [review]
Looks Good to Me

File: ui/components/MesheryApplications.js [hunk 1]
```diff
@@ -481,16 +481,26 @@ function MesheryApplications({
...
-          method : "DELETE",
+          method : "PUT",
+          body : JSON.stringify({ application_data : { id, name, application_file : data }, save : true }),
         },
         () => {
-          console.log("ApplicationFile API", `/api/application/${id}`);
+          console.log("ApplicationFile API", `/api/application/${source_type}`);
           updateProgress({ showProgress : false });
-          enqueueSnackbar(`"${name}" Application deleted`, {
+          enqueueSnackbar(`"${name}" Application Updated Successfully`, {
```
Comment (Chadha93) [inline ui/components/MesheryApplications.js:503]:
```suggestion
          enqueueSnackbar(`"${name}" Application Updated`, {
```

Comment (Chadha93) [reply ui/components/MesheryApplications.js:503]: Please don't use "successfully" in the snackbar messages, please also update the delete snackbar, it should show the app name when deleted. It was there previously.

File: ui/components/Header.js [hunk 2]
```diff
@@ -316,7 +316,7 @@ function K8sContextMenu({
   useEffect(() => {
     setTransformProperty(prev => (prev + (contexts.total_count ? contexts.total_count * 3.125 : 0)))
   }, [])
-
+  console.log(anchorEl);
```
Comment (leecalcote) [inline ui/components/Header.js:319]: Please remove this line.

Comment (leecalcote) [inline ui/components/Header.js:319]:
```suggestion
```

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Pull request merged**